### DISCUSSION TITLE
wezterm can run in computer without GPU?

### DISCUSSION THREAD

**Discussion by r-xiumu** [wezterm/wezterm#2433] [Q&A]
My computer has no GPU.
I download it and installed wezterm, but it can not run.

Is it normal?

**Comment by wez** [comment]
wezterm runs best when there is a GPU. wezterm is able to fall back to CPU based rendering on a number of operating systems.
It's difficult to offer more helpful advice as you didn't share much information. If you'd like to follow up, please open an issue and be sure to provide all the information that is requested by the form: https://github.com/wez/wezterm/issues/new?assignees=&labels=bug&template=bug.yml

**Reply by r-xiumu** [reply]
I report this bug. Thanks! 
https://github.com/wez/wezterm/issues/2436
